server/src/main/java/se/fortnox/reactivewizard/server/ServerConfig.java

```diff
@@ -59,7 +59,7 @@ public int getShutdownTimeoutSeconds() {
         return shutdownTimeoutSeconds;
     }
 
-    public void setShutdownTimeoutMs(int shutdownTimeoutSeconds) {
+    public void setShutdownTimeout(int shutdownTimeoutSeconds) {
```

Comment:
Is it the method or property name that is mapped to the config value? Just to make sure this isn't a breaking change :)

Reply:
Yes this will be a breaking change i think. I will revert it. it might cause problems for people when updating to newer version. But the name is indeed not very good :)

Reply:
I will revert the change, so that no breaking change is introduced.